lib/menu.rb

```diff
@@ -0,0 +1,25 @@
+class Menu  
+  def print_menu
+    format_menu
+  end
+
+  def dishes
```

Comment:
I don't think this method is being used by any of the other classes right now. If it isn't it should be private or as it is a constant I believe already accessible to other internal classes.
If it is there purely for a test it shouldn't be.